### DISCUSSION TITLE
Bulk edit time for photos

### DISCUSSION THREAD

**Discussion by luleyleo** [ente-io/ente#754] [Enhancements]
I have a bunch of photos from a camera which had the wrong year, but the correct date and time set. Now I would like to move all of these photos 20 years into the future, while preserving the time they were shot at and preserving the day / month.

As far as I could tell, the current "fix time" feature only allows me to override everything, which I don't want to do.

**Comment by vishnukvmd** [comment]
>  I would like to move all of these photos 20 years into the future

Sorry, we don't support future timestamps at the moment

**Reply by luleyleo** [reply]
Ah, I wanted to say, move them 20 years into the future, relative to their way too old date. So, from 2003 (what the camera thought it was) to 2023 (what it actually was).

**Reply by luleyleo** [reply]
So this is not about time stamps in the future. Just fixing timestamps from wrongly configured cameras :)

**Reply by vishnukvmd** [reply]
Could you please try opening the image on the mobile app, tap `Info`, and click on the `Edit` icon next to the date and roll the years forward?

**Reply by luleyleo** [reply]
That works, but only for a single image. I want to do this for a hundred images in bulk, each image having its own time and date, but all relative to a wrongly set date (20 years in the past).

**Comment by vishnukvmd** [comment]
Closing in favor of the request @ https://github.com/ente-io/ente/discussions/949. Solving that should take care of this use case as well. Please reopen if you think otherwise!
